README.rst

```diff
@@ -26,3 +26,362 @@ Run `tox` to run the whole test suite:
       lint: commands succeeded
       flake8: commands succeeded
       congratulations :)
+
+
+Usage
+-----
+
+RIOTCtrl provides a python object abstraction of a RIOT device. It's meant
+as a starting point for any serial abstraction on which higher level
+abstractions (like a shell) can be built.
+
+```python
+from riotctrl.ctrl import RIOTCtrl
+
+env = {'BOARD': 'native'}
+# if not running from the application directory the a path must be provided
+ctrl = RIOTCtrl(env=env, application_directory='.')
+# flash the application
+ctrl.make_run(['flash'])
+# run the terminal through a contextmanager
+with ctrl.run_term():
+    ctrl.term.expect('>')       # wait for shell to start
+    ctrl.term.sendline("help")  # send the help command
+    ctrl.term.expect('>')       # wait for the command result to finnish
+    print(ctrl.term.before)     # print the command result
+# run without a contextmanager
+ctrl.start_term()               # start a serial terminal
+ctrl.term.sendline("help")      # send the help command
+ctrl.term.expect('>')           # wait for the command result to finnish
+print(ctrl.term.before)         # print the command result
+ctrl.stop_term()                # close the terminal
+```
```

Comment:
This (and in other places) needs to be restructuredText, not Markdown. Currently, the output [is garbled](https://github.com/RIOT-OS/riotctrl/blob/7421debeb00cfc44b657dc9a7821fa58514a8ecb/README.rst#usage). You can use tools like pandoc to convert the formats.

Reply:
converted, thanks!